ga4gh/exceptions.py

```diff
@@ -180,6 +187,60 @@ def getMessage(self):
             self.callSetId, self.variantSetId)
         return message
 
+
+class DataException(BaseServerException):
+    message = "Faulty data found."
+
+
+class EmptyDirException(DataException):
+    message = "Diretory empty, no VCF/BCF file was found"
+
+
+class MalFormedException(DataException):
```

Comment:
I agree with @jeromekelleher 

In any event, I would object to the name because 'Malformed' is one word, so the camel-casing is wrong